app/src/main/java/com/eveningoutpost/dexdrip/StartNewSensor.java

```diff
@@ -169,7 +175,17 @@ private void startSensorOrAskForG6Code() {
         if (Ob1G5CollectionService.usingCollector() && Ob1G5StateMachine.usingG6()) {
             if (JoH.pratelimit("dex-stop-start", cap)) {
                 JoH.clearRatelimit("dex-stop-start");
-                G6CalibrationCodeDialog.ask(this, this::startSensorAndSetIntent);
+                val transmitterAgeInDays = DexTimeKeeper.getTransmitterAgeInDays(getTransmitterID());
+                val modified = FirmwareCapability.isTransmitterModified(getTransmitterID());
+                val endOfLife = transmitterAgeInDays > 179 || (!modified && transmitterAgeInDays > 99);
+                if (transmitterAgeInDays < 69 || (modified && transmitterAgeInDays < 149)) {
```

Comment:
What is the reason for preferring constants?  I'm not objecting.   I would like to know so that I remember not to repeat it.  Knowing the reason helps me remember.

Reply:
I can think of one.  If we decide to change the value, there will be no need to edit all translations, only the constant.  Whereas if we leave it as a number, changing the value would require changing all translations.  
Is that the benefit?